packages/nvidia_nat_core/src/nat/middleware/defense/defense_middleware_content_guard.py

```
@@ -348,43 +336,33 @@ async def _process_content_safety_detection(
-
-            # Handle output analysis (only output is supported)
-            output = await self._process_content_safety_detection(output, "output", context, original_input=value)
-
-            return output
-
+            # Handle function output analysis
+            original_input = context.original_args[0] if context.original_args else None
+            context.output = await self._process_content_safety_detection(context.output,
+                                                                          func_ctx,
+                                                                          original_input=original_input)
+            return context
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)